sources/packages/api/src/services/education-program-offering/custom-validators/program-allows-offering-delivery.ts

```diff
@@ -44,18 +44,24 @@ class ProgramAllowsOfferingDeliveryConstraint
   }
 
   defaultMessage(args: ValidationArguments) {
-    return `${args.property} has an offering delivery that is not allowed by its program.`;
+    const [propertyDisplayName] = args.constraints;
+    return `${
```

Comment:
Asking for my understanding, why are we choosing between args.constraints and args.property here?
what could be the values here.

Reply:
`args.property` will contain the name of the property itself. In the below example it would be "nameOfTheProperty".
```ts
@SomeValidator()
nameOfTheProperty: string;
```
The idea of having a `propertyDisplayName` is to provide a user-friendly version of "nameOfTheProperty", like "Name of the property" or whatever would be easier for the user.

Reply:
Thanks for for explaining. Now I can understand this piece of code more accurately 
![image](https://user-images.githubusercontent.com/54600590/190229259-52421b2a-7cc5-48f0-8acc-b9ffceae56b7.png)